src/models/cora_module.py

```diff
@@ -111,8 +113,9 @@ def test_step(self, batch: Any, batch_idx: int):
             label_names=LABEL_NAMES
         )
 
-        true_labels = torch.flatten(true_labels).to(torch.device("cuda", 0))
-        true_preds = torch.flatten(true_preds).to(torch.device("cuda", 0))
+        device = self.val_acc.device
```

Comment:
device usage in prediction?

Reply:
The Trainer knows the device and the data in the model will be put on the right device .
What are not parameters need 'to(device)' manually.
> The [LightningModule](https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.core.LightningModule.html#pytorch_lightning.core.LightningModule) knows what device it is on. You can access the reference via self.device. Sometimes it is necessary to store tensors as module attributes. However, if they are not parameters they will remain on the CPU even if the module gets moved to a new device.